modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/fixstainmount/v0.1.0.json

```diff
@@ -0,0 +1,73 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpactsh": "{{base}}/schemas/bbp/core/activity/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/fixstainmount/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/bbp/core/activity/v0.1.0",
+    "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0"
+  ],
+  "shapes": [ {
+    "@id" : "this:FixStainMountShape",
+    "@type" : "sh:NodeShape",
+    "label" : "Fixing, Staining and Mounting activities description",
+    "targetClass" : "nsg:FixStainMount",
+    "nodeKind" : "sh:BlankNodeOrIRI",
+    "and": [
+      {
+        "node": "bbpactsh:ActivityShape"
+      },
+      {
+        "property": [
+          {
+            "path": "nsg:fixationMethod",
+            "name": "Fixation method",
+            "description": "Method used for the fixation of the slice after electrophysiology such as PFA.",
+            "datatype": "xsd:string",
+            "maxCount": 1
+          },
+          {
+            "path": "nsg:stain",
+            "name": "Stain",
+            "datatype": "xsd:string",
+            "description": "Stain used for the detection of the cell labeling compound such as the chromogen DAB.",
+            "maxCount": 1
+          },
+          {
+            "path": "nsg:mountingMedia",
+            "name": "Mounting Media",
+            "datatype": "xsd:string",
+            "description": "Mounting media is the medium that the sample is in while it is being imaged on the microscope such as Xylene or Mowiol.",
+            "maxCount": 1
+          },
+          {
+            "path": "nsg:hadProtocol",
```

Comment:
nsg:hadProtocol is inherited from "bbpactsh:ActivityShape". It should be removed from this schema.